api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java

```diff
@@ -119,6 +119,13 @@ public void started() {
             Context.getAdministrationService().purgeGlobalProperty(defaultDashboard);
         }
 
+        // to replace '/coreapps/clinicianfacing/patient.page?patientId={{patientId}}&amp;app=pih.app.clinicianDashboard'
+        GlobalProperty dashboardUrl = Context.getAdministrationService().getGlobalPropertyObject(CoreAppsConstants.GP_DASHBOARD_URL);
+        if (dashboardUrl != null) {
```

Comment:
Yes. My bad. This was to replace existing Global Property value that has been set by OpenMRS during installation earlier.